frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2119,4 +2119,10 @@ function ReaderFooter:onScreenResize()
     self:resetLayout(true)
 end
 
+function ReaderFooter:onBookmarksChanged()
+    if self.settings.frontlight then
+        self:onUpdateFooter(true)
```

Comment:
Why the dependancy on `settings.frontlight` ?

Reply:
bad copy paste, removed